lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -107,6 +107,25 @@ def self.calculate_next_version(current_version, type_of_bump, snapshot)
         snapshot ? "#{next_version}-SNAPSHOT" : next_version
       end
 
+      def self.detect_bump_type(version_name_a, version_name_b)
+        if version_name_a == version_name_b
+          return :none
+        end
+
+        version_a = Gem::Version.new(version_name_a)
+        version_b = Gem::Version.new(version_name_b)
+
+        same_major = version_a.canonical_segments[0] == version_b.canonical_segments[0]
+        same_minor = version_a.canonical_segments[1] == version_b.canonical_segments[1]
```

Comment:
This would work as long as both versions have 3 canonical segments. Maybe we should validate that? NABD though, since this is internal.